File: 화연/Lv.0/가까운1찾기.js
/**
 * https://school.programmers.co.kr/learn/courses/30/lessons/181898
 * @author HwaYeon
 * @param {number[]} arr 
 * @param {number} idx 
 * @returns 
 */
function solution(arr, idx) {
  for(let i = 0; i < arr.length; i++) {
      if(i + 1 > idx && arr[i] === 1) return i;
  }
  return -1;
}


/**
 * 대충격임!!!! indexOf가 몇번째 인덱스부터 찾아야하는지 구현할 수 있다니!!!!!!
 * includes도 마찬가지!!!
 * @param {number[]} arr 
 * @param {number} idx 
 * @returns 
 */
function solution(arr, idx) {
  return arr.indexOf(1, idx);
}

File: 형민/Lv.0/가까운 1 찾기.java
class Solution {
    public int solution(int[] arr, int idx) {
        int answer = -1;
        
        // idx보다 크면서 배열의 값이 1인 가장 작은 인덱스를 찾아서 반환
        // 그런 인덱스가 없다면, -1을 반환
        for(int i = 0; i < arr.length; i++) {
            if(idx <= i && arr[i] == 1) {
                return i;
            }
        }
        return answer;
    }
    }
